packages/location-management/README.md

```diff
@@ -0,0 +1,10 @@
+# Location Management
+
+This package contains a set of components for managing (create, edit, view, delete) individual location groups/tags.
+
+## Installation
+
+```sh
+
+yarn add @opensrp/location-management
+```
```

Comment:
if possible, should we add a description of the configuration options that can be passed and a code sample

Reply:
@p-netm At this point, there is nothing to add because location management components will be added on top of this skeleton, maybe after each component is added, then the READMe can be updated

Reply:
ok, can we please add that as part of the checklist in the issue describing the work thats adds the components.